watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/supercritical_sludge_to_gas/supercritical_sludge_to_gas.py

```diff
@@ -53,15 +53,18 @@ def main():
 
     results = solve(m)
     assert_optimal_termination(results)
-    display_results(m)
+    # display_reports(m)
 
     add_costing(m)
     initialize_costing(m)
     assert_degrees_of_freedom(m, 0)
     assert_units_consistent(m)
 
     results = solve(m)
-    display_costing(m)
+    assert_optimal_termination(results)
```

Comment:
Note that in #621, @adam-a-a has suggested the use of `check_optimal_termination` instead of `assert_optimal_termination`.